### PR TITLE
[ENG-1969] Documentation and polish for COCO and VOC export converters

### PR DESCRIPTION
**Pull request by nikkigantos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Table of Contents
       * [Reference Interfaces](#reference-interfaces)
       * [Local Development of Labeling Interfaces](#local-development-of-labeling-interfaces)
       * [Installing a Labeling Frontend in labelbox.com](#installing-a-labeling-frontend-in-labelboxio)
+   * [COCO and VOC Exporters](#coco-and-voc-exporters)
```
Comment (euwest) [inline README.md]:
```
[Export Converters ](#coco-and-voc-exporters)
- [COCO](https://github.com/Labelbox/Labelbox/tree/master/exporters/voc-exporter)
- [VOC](https://github.com/Labelbox/Labelbox/tree/master/exporters/coco-exporter)

File: README.md [hunk 2]
```diff
@@ -182,6 +183,11 @@ hosting service on-hand, you can quickly get setup with
 
 ![](https://s3-us-west-2.amazonaws.com/labelbox/labelbox_cloud_deploy.gif)
 
+### COCO and VOC Exporters
+If you need to convert a Labelbox JSON export asset to VOC or COCO file formats, see the README for each directory in this repo:
```
Comment (euwest) [inline README.md]:
```
### Labelbox Export Converters
If you need to convert your project's labels to COCO or VOC format, 
export them in JSON and see the README in either the [COCO](https://github.com/Labelbox/Labelbox/tree/master/exporters/coco-exporter) 
or [VOC](https://github.com/Labelbox/Labelbox/tree/master/exporters/voc-exporter) 
export converter section for your next steps.

File: exporters/coco-exporter/README.md [hunk 3]
```diff
@@ -3,17 +3,26 @@
...
-To get started if you do not already have Docker you'll want to download and install Docker desktop. You can do so here: [docker-hub](https://hub.docker.com/). 
-
-Once you have installed and signed into Docker you may run the converter inside a docker container. We HIGHLY recommend you use the Docker container to run the converter script `main.py`.
-The docker container provides an isolated environment with the correct dependencies installed, making it easier to setup than even a pipenv.
+First clone the repository to access the converter script aka:
+```sh
+git clone https://github.com/Labelbox/Labelbox.git
+```
 
+To get started if you do not already have Docker you'll want to download and install Docker desktop. You can do so here: [docker-hub](https://hub.docker.com/).
```
Comment (euwest) [inline exporters/coco-exporter/README.md]:
```
To get started you'll want to download and install Docker on your desktop. 
Follow the installation and setup instructions at [docker-hub](https://www.docker.com/products/docker-desktop).

File: exporters/coco-exporter/README.md [hunk 4]
```diff
@@ -3,17 +3,26 @@
...
-The docker container provides an isolated environment with the correct dependencies installed, making it easier to setup than even a pipenv.
+First clone the repository to access the converter script aka:
+```sh
+git clone https://github.com/Labelbox/Labelbox.git
+```
 
+To get started if you do not already have Docker you'll want to download and install Docker desktop. You can do so here: [docker-hub](https://hub.docker.com/). 
 
-To run the converter in docker, from this directory run the command (where `${PATH_TO_INPUT}` is the location of your Labelbox JSON export file on your local machine)
+Once you have installed Docker, you may run the converter inside a docker container. To do so simply run the following command from this directory (where `${PATH_TO_INPUT}` is the location of your Labelbox JSON export file on your local machine).
```
Comment (euwest) [inline exporters/coco-exporter/README.md]: >Once you have Docker installed and running you can use the export converter. In your terminal navigate to this directory (i.e. `Labelbox/exporters/coco-exporter`) and run:
>```sh
># e.g. if your json export file is named "export.json" and is in your Downloads directory
>
>make run-local-export EXPORT_PATH=~/Downloads/export.json
>```

Comment (nikkigantos) [reply exporters/coco-exporter/README.md]: yeah i always struggle with how to best communicate variables in command line docs. i dig it :)

**Pull request merged**